.github/workflows/archive/cpdb_spending.yml

```diff
@@ -37,7 +37,8 @@ jobs:
       - uses: actions/checkout@v3
 
       - name: install dependencies ...
-        run: ../bash/docker_container_setup.sh
+        working-directory: ./
```

Comment:
when no bash defaults are set for the job, is `working-directory: ./` needed?

Reply:
This does have a default working directory, though that wasn't updated! I'll do another pass on my local files

Reply:
Fixed